numpy/lib/twodim_base.py

```diff
@@ -172,21 +245,33 @@ def rot90(m, k=1):
     >>> np.rot90(m, 2)
     array([[4, 3],
            [2, 1]])
+    >>> m = np.arange(8).reshape((2,2,2))
+    >>> np.rot90(m, 1, (1,2))
+    array([[[1, 3],
+            [0, 2]],
+
+          [[5, 7],
+           [4, 6]]])
 
     """
     m = asanyarray(m)
+
     if m.ndim < 2:
         raise ValueError("Input must >= 2-d.")
```

Comment:
You should A) add a check for `len(axes) == 2` (and a test that it raises appropriately), B) change this to check that neither axis is `>= ndim`.

Also, if both axes are the same you may want to either raise or return the input array. Either way, you should document how you do it.